dashboard/src/components/LoadingWrapper/LoadingWrapper.tsx

```diff
@@ -1,7 +1,15 @@
 import * as React from "react";
+
 import LoadingSpinner from "../LoadingSpinner";
+import LoadingPlaceholder from "./LoadingPlaceholder";
```

Comment:
Maybe this should be a little bit more specific?

`LoadingTextPlaceholder`?

Reply:
I feel like the "placeholder" is a style in itself, just search for "placeholder loader" on Google and you come up with exactly what this is: https://www.google.com/search?q=placeholder+loader&oq=placeholder+loader&aqs=chrome.0.69i59j69i60j0l4.3666j1j7&sourceid=chrome&ie=UTF-8.